AUTHORS.md

```diff
@@ -538,7 +538,7 @@ and provided thanks to the work of the following contributors:
 * [Drew Schuster](mailto:dtschust@gmail.com)
 * [Dryusdan](mailto:dryusdan@dryusdan.fr)
 * [Eai](mailto:eai@mizle.net)
-* [Eashwar Ranganathan](mailto:eranganathan@lyft.com)
+* [Eashwar Ranganathan](mailto:eashwar@eashwar.com)
```

Comment:
I believe this file is periodically regenerated (at release) and not meant to be edited ... I think it pulls from github api to grab emails, so if you change there it will eventually change here.

Reply:
Ah, makes sense. Unless you feel strongly about it I'll probably leave this change here anyway. If it gets overwritten again, oh well.